seabird_chat_ingest.proto

```diff
@@ -0,0 +1,144 @@
+syntax = "proto3";
+
+option go_package = ".;pb";
+
+import "common.proto";
+
+package seabird;
+
+// Note that all of the types in this file end with ChatRequest or end with
+// ChatEvent to prevent namespace conflicts between the main seabird.proto file.
+
+// SendMessageChatRequest requests for a message to be sent to a given channel.
+// Either SuccessChatEvent or FailedChatEvent SHOULD be returned in response if
+// possible. This MUST NOT cause any chat events from the bot to appear in the
```

Comment:
`SHOULD be returned...if possible.` -- what happens if not, and why is it beneficial? What will core do with them?

Reply:
Core will return a timeout error if it isn’t received in time

Reply:
I'd still like to see each of the comments about failure characteristics pulled out and added above this